From 3f44c0f656edd3a97ea3d71d0c68551434db5d69 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Wed, 3 Apr 2019 02:19:02 +0000
Subject: [PATCH 2/3] spp_primary: remove declaration in for loop

For some environment, declaration in for loop cause an error. This
update is to move it from.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/primary/init.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/primary/init.c b/src/primary/init.c
index 28ad0a7..60e5c47 100644
--- a/src/primary/init.c
+++ b/src/primary/init.c
@@ -111,6 +111,7 @@ init(int argc, char *argv[])
 	const struct rte_memzone *mz;
 	uint16_t count, total_ports;
 	char log_msg[1024] = { '\0' };  /* temporary log message */
+	int i;
 
 	/* init EAL, parsing EAL args */
 	retval = rte_eal_init(argc, argv);
@@ -169,7 +170,7 @@ init(int argc, char *argv[])
 		lcore_id_used[lcore_id] = 1;
 	}
 	sprintf(log_msg, "Used lcores: ");
-	for (int i = 0; i < RTE_MAX_LCORE; i++) {
+	for (i = 0; i < RTE_MAX_LCORE; i++) {
 		if (lcore_id_used[i] == 1)
 			sprintf(log_msg + strlen(log_msg), "%d ", i);
 	}
-- 
2.7.4

